src/App/Fig.purs

```diff
@@ -186,8 +188,8 @@ drawFig divId fig = do
    redraw = (_ $ fig { ι = ι }) >>> drawFig divId
 
 drawFile :: File × String -> Effect Unit
-drawFile (File file × src) =
-   addEditorView (codeMirrorDiv file) >>= drawCode src
+drawFile (File filepath × src) =
+   addEditorView (codeMirrorDiv (toFileName filepath)) >>= drawCode src
```

Comment:
Bit unsure as to the need for `toFileName` -- we seem to be throwing some information away here. Shouldn't we just pass the correct file name to `drawFile` in the first place?